test_conformance/events/test_event_dependencies.cpp

```diff
@@ -435,9 +443,10 @@ int test_event_enqueue_wait_for_events_run_test(
 
     error = clEnqueueReadBuffer(queues[0], data, CL_TRUE, 0,
                                 TEST_SIZE * sizeof(cl_int), values, 1,
-                                &event[event_count], NULL);
+                                &last_loop_event, NULL);
 
     test_error(error, "clEnqueueReadBuffer failed");
+    last_loop_event.reset();
```

Comment:
Shouldn't need this manual reset, will happen when `last_loop_event` goes out of scope at the end of the function

Reply:
thank you , done